libraries/radpdfprocessing/model/form.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Form
+page_title: Form
+description: Form
+slug: radpdfprocessing-model-form
+tags: form
+published: True
+position: 6
+---
+
+# Form
+
+
+
+__Form__ is a content element, which contains a form source and represents a Form XObject. The Form XObjects enables you to describe objects (text, images, vector elements, etc.) within a PDF file and reuse this content among the document.
```

Comment:
If there is an online resource describing the Form XObjects we can link it here.